src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography as TypographyMUI, TypographyProps } from '@mui/material';
+import '../../config/fonts/styles.css';
+import {
+  ThemeProvider,
+  responsiveFontSizes,
+  createTheme,
+} from '@mui/material/styles';
+
+export const Typography = (props: TypographyProps) => {
+  const {
+    children,
+    variant,
+    align,
+    gutterBottom,
+    noWrap,
+    paragraph,
+    fontFamily,
+  } = props;
+
+  let theme = createTheme();
+  theme = responsiveFontSizes(theme);
+
+  return (
+    <ThemeProvider theme={theme}>
+      <TypographyMUI
+        variant={variant}
+        align={align}
+        gutterBottom={gutterBottom}
+        noWrap={noWrap}
+        paragraph={paragraph}
+        sx={{ fontFamily: fontFamily }}
+      >
+        {children}
+      </TypographyMUI>
+    </ThemeProvider>
+  );
+};
+
+Typography.propTypes = {
+  children: PropTypes.string.isRequired,
+  variant: PropTypes.oneOf([
+    'h1',
+    'h2',
+    'h3',
+    'h4',
+    'h5',
+    'h6',
+    'subtitle1',
+    'subtitle2',
+    'body1',
+    'body2',
+    'caption',
+    'button',
+    'overline',
+    'inherit',
+  ]),
+  fontFamily: PropTypes.oneOf(['DM Mono', 'Arizona Serif', 'default']),
+  align: PropTypes.oneOf(['center', 'inherit', 'justify', 'left', 'right']),
+  gutterBottom: PropTypes.bool,
+  noWrap: PropTypes.bool,
+  paragraph: PropTypes.bool,
+};
+
+Typography.defaultProps = {
+  children: 'Hello World',
+  variant: 'body1',
+  align: 'left',
+  gutterBottom: false,
+  noWrap: false,
+  paragraph: false,
+  fontFamily: 'default',
```

Comment:
I think this should be Arizona Serif, since that's our default font.

Reply:
So if I understand correctly Arizona Serif is our default font in all products, right? I thought that maybe it's in some, so then it would make quite tricky all the time typing in different font family

Reply:
And fix for that